src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: string;
```

Comment:
make type for shape and use in all places
type Shape = 'triangle' | 'circle' | 'rectangle';

```suggestion
  shape: Shape;
```